src/service/get-words.js

```diff
@@ -1,9 +1,10 @@
-import { WORDS_COUNT } from '../config/config';
+import { httpService } from './http-service';
 import { getRandomIndexInRange } from '../helpers';
-import { words } from '../data/data';
 
-const getWords = () => {
-    const index = getRandomIndexInRange(WORDS_COUNT);
+const getWords = async () => {
```

Comment:
regarding `words` and `allWords`, I think too much is going on and we need to simplify

some notes about terms/names used in the app:
`words` - I think this is array of strings, formerly known as `correctWords`
`allWords` - this is clear, array of arrays of strings, basically all possible words that you can see in the game if you play million times :)

these two terms shouldn't be confused and when naming things it's important to be consistent

but `httpService.getWords` fetches all words, it's a bit confusing

also, `getAllWords` action calls `axios.get('words.json')` directly without `httpService`, there should be one place in code where fetching all words is done, perhaps some unused code is left, if this is the case please clean it up, it would be easier to read PR

fetching all words should be done once in app's lifetime
fetching words (array of strings) should be done once per single game
I imagine these are two separate actions

Reply:
hm, `service` directory is not in use right now so I didn't change any names yet. i had deleted this directory but you asked me to restore it, remember?

Reply:
I don't have a full memory of past comments :(
the point is this PR in current shape has many good parts but is in need of a cleanup (e.g. removal of dead code or starting using it)
I don't know what to think when I read unused code, is it something you want to use? is it something you don't like? is it something you don't understand?

do you think you can spend a little more time and tidy things up? I think you are very close! I've tried to give you specific advice, if you have a question pleask ask
now I don't know how to help you, I don't want to provide ready solution